cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -114,3 +121,28 @@ def test_order_catalog_item(catalog_item, request):
         fail_func=requests.reload, num_sec=1400, delay=20)
     assert 'Provisioned Successfully' in row.last_message.text
     DefaultView.set_default_view("Configuration Management Providers", "List View")
+
+
+@pytest.mark.tier(2)
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.6')
+@pytest.mark.ignore_stream("upstream")
+@pytest.mark.meta(blockers=[BZ(1420179, forced_streams=["5.6", "5.7", "upstream"])])
+def test_retire_ansible_service(catalog_item, request):
+    """Tests order catalog item
+    Metadata:
+        test_flag: provision
+    """
+    catalog_item.create()
+    service_catalogs = ServiceCatalogs(catalog_item.name)
+    service_catalogs.order()
+    logger.info('Waiting for cfme provision request for service %s', catalog_item.name)
+    row_description = catalog_item.name
+    cells = {'Description': row_description}
```

Comment:
**OPTIONAL**
Since this is only used here just put `catalog_item.name` in the dictionary directly